LDAR_Sim/src/file_processing/output_processing/multi_simulation_visualizations.py

```diff
@@ -27,6 +28,67 @@
 from matplotlib import ticker, lines
 
 from file_processing.output_processing import output_constants, output_utils
+import numpy as np
+
+Y_AXIS_LABEL = "Likelihood of Occurrence (%)"
```

Comment:
Can these get moved to wherever we are keeping the rest of the constants for outputs?